source/ДЗ/Циклы.1/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Циклы._1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+           
+            Console.Write("Введите конечное число дюймов:  ");
+            double inch = double.Parse(Console.ReadLine());
+            Console.Write("Введите шаг :  ");
+            double step = double.Parse(Console.ReadLine());
+            for (double i = 1; i <= inch; i += step)
+            {
+                Console.Write(i + "in. = "); Console.Write(i * 2.54 + "cm.\n");
```

Comment:
Сокращенно дюйм обозначают _in_ пробел без точки) или " (двойной штрих). Сантиметры тоже отделяются пробелом и после них не нужно ставить точку.
```suggestion
                Console.Write(i + " in = "); Console.Write(i * 2.54 + " cm\n");
```